Dizzy/Screens/Map/MapVM.swift

```diff
@@ -9,25 +9,35 @@
 import Foundation
 
 protocol MapVMType {
-    var currentLocation: Observable<Location?> { get set }
+
     var currentAddress: Observable<Address?> { get set }
+    var selectedLocation: Observable<Location?> { get set }
+    var marks: Observable<[Marks?]> { get set }
+    var showCancelLocationSelection: Observable<Bool> { get set }
+
     var delegate: MapVMDelegate? { get set }
     
-    func getAllMarks() -> [Marks]
+    func searchButtonPressed()
+    func didSelect(place: PlaceInfo)
+    func returnMapToInitialState()
```

Comment:
i think better name will be `resetMapToInitialState`

Reply:
done